.circleci/scripts/test-run-e2e-timeout-minutes.ts

```diff
@@ -2,7 +2,7 @@ import { filterE2eChangedFiles } from '../../test/e2e/changedFilesUtil';
 
 const changedOrNewTests = filterE2eChangedFiles();
 
-// 20 minutes, plus 3 minutes for every changed file, up to a maximum of 30 minutes
-const extraTime = Math.min(20 + changedOrNewTests.length * 3, 30);
+// 20 minutes, plus 3 minutes for every changed file, up to a maximum of 60 minutes
+const extraTime = Math.min(20 + changedOrNewTests.length * 3, 60);
```

Comment:
Curious why this is required?

Reply:
Hi @pedronfigueiredo , we have the label "skip-e2e-quality-gate" that can bypass the check, it should be used with a valid reason to avoid abuse usage.
Also, we are trying to migrate to page object modal, since this PR is big enough, maybe we can address it in separate PR? thanks!